### PR TITLE
Add regression SVM to docs

### PR DESCRIPTION
**Pull request by PetrovKP**

### CODE REVIEW

File: docs/source/includes/svm/svm-introduction.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@
 .. * limitations under the License.
 .. *******************************************************************************/
 
-Support Vector Machine (SVM) :capterm:`classification` is among popular classification
+Support Vector Machine (SVM) :capterm:`classification` and :capterm:`regression` is among popular
```
Comment (averbukh) [inline docs/source/includes/svm/svm-introduction.rst:17]: is -> are ?

Comment (PetrovKP) [reply docs/source/includes/svm/svm-introduction.rst]: thanks

File: docs/source/onedal/algorithms/svm/svm.rst [hunk 2]
```diff
@@ -33,28 +33,63 @@ Mathematical formulation
...
 --------
 
-Given :math:`n` feature vectors :math:`x_1 = (x_{11}, \ldots, x_{1p}), \ldots, 
-x_n = (x_{n1}, \ldots, x_{np})` of size :math:`p` and a vector
-of class labels :math:`y = (y_1, \ldots, y_n)`, where :math:`y_i \in \{-1, 1\}` 
-describes the class to which the feature vector :math:`x_i` belongs, the problem is to build a 
-two-class Support Vector Machine (SVM) classifier.
+Given :math:`n` feature vectors :math:`X=\{x_1=(x_{11},\ldots,x_{1p}),\ldots,
+x_n=(x_{n1},\ldots,x_{np})\}` of size :math:`p`,
+their non-negative observation weights :math:`W=\{w_1,\ldots,w_n\}`
```
Comment (outoftardis) [inline docs/source/onedal/algorithms/svm/svm.rst:38]:
```suggestion
their non-negative observation weights :math:`W=\{w_1,\ldots,w_n\}`,
```

File: docs/source/onedal/algorithms/svm/svm.rst [hunk 3]
```diff
@@ -33,28 +33,63 @@ Mathematical formulation
...
+    where :math:`C` is the upper bound of the coordinates of the vector :math:`\alpha`,
+    :math:`Q` is a symmetric matrix of size :math:`2n \times 2n`
+    with :math:`Q_{ij} = y_i y_j K(x_i, x_j)`, and :math:`K(x,y)` is a kernel function.
+    Vectors :math:`s` and :math:`z` for the regression problem are formulated according
+    to the following rule:
+
+    .. math::
+      \begin{cases}
+         z_i = +1, s_i = \epsilon - y_i, i \leq n \\
+         z_i = -1, s_i = \epsilon + y_i, n < i \leq 2n
```
Comment (outoftardis) [inline docs/source/onedal/algorithms/svm/svm.rst:83]:
```suggestion
         z_i = +1, s_i = \epsilon - y_i, & i \leq n \\
         z_i = -1, s_i = \epsilon + y_i, & n < i \leq 2n
```

**Review by outoftardis — APPROVED** [review]

**Pull request merged**